packages/flutter/lib/src/cupertino/nav_bar.dart

```diff
@@ -2292,9 +2309,9 @@ class _ActiveSearchableBottom extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return Padding(
-      padding: const EdgeInsets.only(left: _kNavBarEdgePadding),
+      padding: const EdgeInsets.only(left: _kNavBarEdgePadding, bottom: _kNavBarBottomPadding),
```

Comment:
Should we be using `left`? Or should we be using `start` for ltr/rtl localization reasons. I believe other parts of the navbar use `start` and `end`. I haven't checked, but I assume the native search bar moves when put into RTL mode.